tests/test_game_logic.py

```
@@ -92,61 +92,51 @@ def test_create_grid_returns_deep_copy():
+    """右キー: 正常な移動 + 右端での制限"""
     locked = empty_grid()
     grid = create_grid(locked)
 
-    # 下に移動できることを確認
+    # 正常な移動: 右に移動できることを確認
+    mino = DummyMino(3, 5, O)
+    _, moved_mino = keyOperation(True, pygame.K_RIGHT, grid, mino)
+    assert moved_mino.x == 4
+
+    # 右端制限: 右端では移動できないことを確認
+    # Oミノは相対位置(2,1)から(2,2)まで占有するので、x=GRID_WIDTH-2が実質的な右端
```

Ruff (0.14.10)

Comment contains ambiguous `ノ` (KATAKANA LETTER NO). Did you mean `/` (SOLIDUS)?

(RUF003)